woocommerce/woocommerce-ios
# [Manual Taxes] Tax lines UI

Closes: #10512 


## Description

With this PR we show all the tax lines in the editable order details. Each line has the tax name, the code, the rate percentage, and the total value. Please note that some taxes can be applicable to only some taxes and not all, so the percentage is not calculated based on the order total.

## Testing instructions


Enable taxes in `wp-admin/admin.php?page=wc-settings&tab=general`
Add a tax rate, e.g. Standard `/wp-admin/admin.php?page=wc-settings&tab=tax&section=standard`
Change the value of "Calculate tax based on" in `wp-admin/admin.php?page=wc-settings&tab=tax` to shop base address

1.  Go to orders
2. Add products to the order. Make sure that the products you add to the order are taxable and that the Tax class matches the tax you added.
3. The tax lines are shown in the editable order details.

You can play with different tax rates/classes (see video below). As mentioned above, an order can contain multiple tax line applicable to different products.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/1864060/0cf85107-10c5-4bc6-842a-c60f7d305867

---
